src/org/spdx/rdfparser/license/License.java

```diff
@@ -604,7 +631,7 @@ public void setDeprecated(boolean deprecated) {
 	public AnyLicenseInfo clone() {
 		try {
 			SpdxListedLicense retval = new SpdxListedLicense(this.getName(), this.getLicenseId(),
-					this.getLicenseText(), this.getSeeAlso(), this.getComment(),
+					this.getLicenseText(), this.getSeeAlso(), this.getSeeAlsoDetails(), this.getComment(),
```

Comment:
Perhaps we should rename the xxxSeeAlsoDetails to xxxCrossRef to be consistent.

Reply:
Corrected